File: HarvardX CS50 - Intro to Programming/craigpauga-cs50-2019-x-sentimental-mario-less/mario.py
from sys import argv

# initialize height
height = input("Height?\n")

# continue until the user gives positive int
while (not height.isnumeric()) or int(height) <= 0 or int(height) >= 9:

    height = input("Height?\n")

# Loop through height
height = int(height)
for i in range(1, height+1):
    for j in range(1, height+1):
        if j > height - i:
            print("#", end='')
        else:
            print(" ", end='')
    print('')

File: HarvardX CS50 - Intro to Programming/craigpauga-cs50-2019-x-mario-less/mario.c
// Craig Pauga
// 15 May 2019
// Problem Set 1

//Create # pyramid from Mario
#include <cs50.h>
#include <stdio.h>

//Declaring function
void mario(void);

//Create # pryamid from Mario
int main(int argc, string argv[])
{
    mario();
}

void mario(void)
{
    //Initialize height
    int n;
    
    //Obtain user input
    do
    {
        n = get_int("Height: ");            
    } 
    while (n <= 0 || n > 8);
    
    //Outer Loop
    for (int i = 1; i <  n + 1; i++)
    {
        //Inner Loop        
        for (int j = 1; j < n + 1; j++)
        {
            //Check if space or #
            if (j > n - i)
            {
                printf("#");
            } 
            else
            {
                printf(" "); 
            }
             
        }
        printf("\n");
    }
}
